Code/Framework/AzFramework/AzFramework/Application/Application.cpp

```diff
@@ -67,11 +68,13 @@
 #include <AzFramework/Terrain/TerrainDataRequestBus.h>
 #include <AzFramework/Viewport/ScreenGeometry.h>
 #include <AzFramework/Visibility/BoundsBus.h>
-#include <AzCore/Console/Console.h>
 #include <AzFramework/Viewport/ViewportBus.h>
-#include <GridMate/Memory.h>
 #include <AzFramework/Physics/HeightfieldProviderBus.h>
 
+#if !defined(_RELEASE)
+#include <AzNetworking/Framework/NetworkingSystemComponent.h>
+#endif
+
```

Comment:
This can be removed now.
The NetworkingSystemComponent should be registering itself through it's AzNetworkingModule constructor

Reply:
Ah, missed this. Thanks!